src/components/books/book-item.tsx

```diff
@@ -20,16 +20,16 @@ export function BookItem(props: BookItemProps) {
     <Card className="shadow-md transition-shadow duration-300 hover:shadow-lg">
       <CardContent className="p-4 md:p-6">
         <div className="flex flex-col gap-4 md:flex-row md:gap-6">
-          <div className="mx-2 w-[130px]">
-            <AspectRatio ratio={1 / 2} className="flex items-center">
+          <div className="mx-2 w-full md:w-1/4">
```

Comment:
Seems like a revert? Why?

Reply:
This was taken from previous book-item the logic wasn't working on some e books and the black contrast was getting messed up for dark theme